### PR TITLE
www-client/elinks: various fixes

### PR DESCRIPTION
**Pull request by doughdemon**

### CODE REVIEW

**Comment by jonasstein** [comment]
I suggest to bump to EAPI=6 for all ebuilds directly, while at it. Here some hints: https://devmanual.gentoo.org/ebuild-writing/eapi/
https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/

File: www-client/elinks/elinks-0.12_pre6-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+src_install() {
+	emake V=1 DESTDIR="${D}" install
+
+	insinto /etc/elinks
+	doins "${WORKDIR}"/elinks.conf
+	newins contrib/keybind-full.conf keybind-full.sample
+	newins contrib/keybind.conf keybind.conf.sample
+
+	dodoc AUTHORS BUGS ChangeLog INSTALL NEWS README SITES THANKS TODO doc/*.*
```
Comment (jonasstein) [inline www-client/elinks/elinks-0.12_pre6-r3.ebuild]: nice work! One more thing: You can put these in the DOCS= variable.

File: www-client/elinks/elinks-0.12_pre6-r3.ebuild [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	einfo "You may want to convert your html.cfg and links.cfg of"
+	einfo "Links or older ELinks versions to the new ELinks elinks.conf"
+	einfo "using /usr/share/doc/${PF}/contrib/conv/conf-links2elinks.pl"
+	einfo
+	einfo "Please have a look at /etc/elinks/keybind-full.sample and"
+	einfo "/etc/elinks/keybind.conf.sample for some bindings examples."
+	einfo
+	einfo "You will have to set your TERM variable to 'xterm-256color'"
+	einfo "to be able to use 256 colors in elinks."
+	echo
```
Comment (jonasstein) [inline www-client/elinks/elinks-0.12_pre6-r3.ebuild]: I would remove the echo here.

File: www-client/elinks/elinks-0.12_pre6-r3.ebuild (hunk 1 above)
Comment (doughdemon) [reply www-client/elinks/elinks-0.12_pre6-r3.ebuild]: The "doc/*.*" cannot directly be put into DOCS, and putting instead "doc/." into docs would install also all  sub-directories of "doc" including "doc/man".

**Pull request closed without merging**